server/src/setup/cache/mock-cache.ts

```diff
@@ -0,0 +1,31 @@
+import { AuthCodeRepo } from '../../modules/users/infra/repos/auth-code-repo/auth-code-repo'
+import { MockAuthCodeRepo } from '../../modules/users/infra/repos/auth-code-repo/implementations/mock-auth-code-repo'
+import { AuthCodeEntity } from '../../shared/infra/cache/entities/auth-code-entity'
+import { Repos, Cache } from './types'
+
+
+interface MockEntities {
+  authCodes?: Array<AuthCodeEntity>
+}
+
+interface MockRepos extends Repos {
+  authCode: AuthCodeRepo
+}
+
+const setupMockRepos = (entities: MockEntities): MockRepos => {
+  return {
+    authCode: new MockAuthCodeRepo(entities.authCodes)
+  }
+}
+
+interface MockDB extends Cache {
```

Comment:
should this be `MockCache`?

Reply:
Don't think so - just like the DB mock, it should extend the same base interface (in this case `Cache` instead of `DB`).